### PR TITLE
fix: a/b grouping logic, update hottest group b view param

### PR DESCRIPTION
**Pull request by Carolusian**
- Take `admin` user into account when grouping users
- Update decay factor to 0.8 for hottest articles

### CODE REVIEW

File: src/common/utils/getViewer.ts [hunk 1]
```diff
@@ -47,6 +49,10 @@ export const getUserGroup = ({
   } catch (error) {
     logger.error(error)
   }
+
+  if (role === USER_ROLE.admin) {
+    return 'b'
```
Comment (robertu7) [inline src/common/utils/getViewer.ts:54]: any related discussion on this change?

**Review by robertu7 — APPROVED** [review]

File: src/common/utils/getViewer.ts (hunk 1 above)
Comment (Carolusian) [reply src/common/utils/getViewer.ts:54]: Yes, @jeremyOk proposed this at wrote the logic in the SQL which is used in migrations: https://github.com/thematters/matters-server/pull/1538/files#diff-b726e1a846dbc8b25876ac766b0ceaa41f4983702650e221ea1a80e84a876a11R23

**Review by devformatters — APPROVED** [review]
Go. 🚟

**Review by guoliu — APPROVED** [review]

**Comment by Carolusian** [comment]
This PR will be closed. As the cookie forwarding mechanism updated

**Pull request closed without merging**